src/components/Multiselect/Multiselect.vue

```diff
@@ -19,6 +19,88 @@
   - along with this program. If not, see <http://www.gnu.org/licenses/>.
   -
   -->
+<docs>
+## Multiselect
+We're wrapping the awesome vue-multiselect library to add our own styling and default props/methods
+You can use all the properties from https://vue-multiselect.js.org that are not declared/overrided here.
+
+### Simple examples
+```vue
+<template>
+	<div class="wrapper">
+		<Multiselect v-model="value1" :options="options" />
+		<Multiselect v-model="value2" :options="options" :multiple="true" />
+	</div>
+</template>
+
+<script>
+import Multiselect from '../index'
+export default {
+	data() {
+		return { value1: 2, value2: [2], options: [0, 1, 2, 3, 4] }
+	}
+}
+</script>
+```
+
+### Limit with automated tooltip
+```vue
+<template>
+	<Multiselect v-model="value"
+		:options="options" :multiple="true"
+		:tag-width="80" />
+</template>
+
+<script>
+import Multiselect from '../index'
+export default {
+	data() {
+		return {
+			value: ['eirmod', 'et', 'magna', 'invidunt', 'tempor'],
+			options: ['Consetetur', 'sadipscing', 'elitr', 'sed',
+				'diam', 'nonumy', 'eirmod', 'tempor', 'invidunt',
+				'ut', 'labore', 'et', 'dolore', 'magna', 'aliquyam', 'erat']
+		}
+	}
+}
+</script>
+```
+
+### User layout
+```vue
+<template>
+	<Multiselect v-model="value" :options="formatedOptions"
+		label="label" track-by="id"
+		:user-select="true" />
+</template>
+
+<script>
+import Multiselect from '../index'
+export default {
+	data() {
+		return {
+			value: { id:2,label:'user 1' },
+			options: ['admin', 'user1', 'user2', 'guest', 'group1']
```

Comment:
Minor issue, when selecting an option in the demo then, it doesn't get shown in the field as the value doesn't have a label property.

Reply:
Thanks :)